### PR TITLE
Rename email notifications attribute email to recipients

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

**Comment by vova07** [comment]
please review

**Comment by vova07** [comment]
test this please

File: spec/definitions/EmailNotifications/EmailNotification.yaml [hunk 1]
```diff
@@ -42,16 +42,19 @@ properties:
...
           format: email
-        email:
-          type: string
-          description: Specified email or placeholder
+        recipients:
+          type: array
+          description: The notification's list of recipient emails or placeholders.
+          minItems: 1
+          items:
+            type: string
```
Comment (2e3s) [inline spec/definitions/EmailNotifications/EmailNotification.yaml]: `format: email`?

Comment (2e3s) [reply spec/definitions/EmailNotifications/EmailNotification.yaml]: or placeholders can be different string, right?

Comment (vova07) [reply spec/definitions/EmailNotifications/EmailNotification.yaml]: nope!
It can be an email or a placeholder.

Comment (vova07) [reply spec/definitions/EmailNotifications/EmailNotification.yaml]: Yes, a placeholder is not a valid email.

**Pull request merged**